Sources/CLibMongoC/include/CLibMongoC_mongoc-config.h

```diff
@@ -44,11 +39,11 @@
 #  define MONGOC_ENABLE_SSL_SECURE_TRANSPORT 0
 #  define MONGOC_ENABLE_CRYPTO_LIBCRYPTO 1
 #  define MONGOC_ENABLE_SSL_OPENSSL 1
-#  define MONGOC_HAVE_ASN1_STRING_GET0_DATA 1
-#endif
-
-#if MONGOC_ENABLE_SSL_OPENSSL && OPENSSL_VERSION_NUMBER < 0x10100000L
-#  define MONGOC_HAVE_ASN1_STRING_GET0_DATA 0
+#  if OPENSSL_VERSION_NUMBER < 0x10100000L
+#    define MONGOC_HAVE_ASN1_STRING_GET0_DATA 1
```

Comment:
I think these defines need to be flipped actually.

Reply:
thanks, fixed